plugins/esi/combo_handler.cc

```diff
@@ -136,6 +138,24 @@ struct InterceptData {
   ~InterceptData();
 };
 
+/*
+ * This class is responsible for keeping track of and processing the various
+ * Cache-Control values between all the requested documents
+ */
+struct CacheControlHeader {
+  // Update the object with a document's Cache-Control header
+  void update(TSMBuffer bufp, TSMLoc hdr_loc);
+
+  // Return the Cache-Control for the combined document
+  string generate() const;
+
+  // Cache-Control values we're keeping track of
+  int _max_age    = 315360000; // max value (10 years)
```

Comment:
Actually I believe the max value should be 1 year. According to [RFC 2616](https://www.ietf.org/rfc/rfc2616.txt), 
```
To mark a response as "never expires," an origin server sends an
Expires date approximately one year from the time the response is
sent. HTTP/1.1 servers SHOULD NOT send Expires dates more than one
year in the future.
```